virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,18 +1,26 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to an external network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+{VirtProductName} is installed with a default internal `pod` network. You update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration] depending on the type of network to which you want to connect.
+
+[IMPORTANT]
+====
+When connecting to an external network, it is essential to first create a _Linux bridge_. Until you create a Linux bridge, devices such as network interface card (NICs) are not available because your external network is not yet exposed. To expose your external network and make such devices available, you first need to create a Linux bridge.
```

Comment:
If you use my recommendation in the previous comment, you do not need this note.

Reply:
Just a note on the Important admonition.....

I think we are overloaded on admonitions in general in our doc set (whose doc set isn't?) but in this case I think an "Important" does have value. In general, most authoring environments don't offer that admonition option, or block it (most DITA environments intentionally block it) on the basis that "if it's not important why would you write it?" Ironically, the more admonitions we use gives rise to a need to call out the "really important" ones........ But I'll defer to your judgement and use the wording you recommend above. Up to you if we want to use an Important.

Reply:
It's deleted in this latest version.